<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Blitz Image Resizer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #37474F;
border: none;
</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>301</x>
      <y>9</y>
      <width>301</width>
      <height>171</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">

QTabWidget::pane { /* The tab widget frame */
    border: none;
background: #263238;
}

QTabWidget::tab-bar {
}

/* Style the tab using the tab sub-control. Note that
    it reads QTabBar _not_ QTabWidget */
QTabBar::tab {
    background: #333;
    border-bottom-color: #C2C7CB; /* same as the pane color */
    min-width: 8ex;
    padding: 8px;
    color: #FFFFFF;
}

QTabBar::tab:selected, QTabBar::tab:hover {
    background: #000000;
}

QTabBar::tab:selected {
    background: #80CBC4;
}

QTabBar::tab:!selected {
    margin-top: 2px; /* make non-selected tabs look smaller */
}</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="usesScrollButtons">
     <bool>false</bool>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Properties</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_Width">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>54</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: none;

color: black;</string>
      </property>
      <property name="text">
       <string>320</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Height">
      <property name="geometry">
       <rect>
        <x>101</x>
        <y>54</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: none;

color: black;</string>
      </property>
      <property name="text">
       <string>220</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>28</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string>background: none;

color: white;</string>
      </property>
      <property name="text">
       <string>Width:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>101</x>
        <y>22</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string>background: none;

color: white;</string>
      </property>
      <property name="text">
       <string>Height:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_aspectRatio">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>93</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string>background: none;

color: white;</string>
      </property>
      <property name="text">
       <string>Maintain Aspect Ratio</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Watermark</string>
     </attribute>
     <widget class="QCheckBox" name="checkBox_watermark">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>12</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white; font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Enabled</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_WatermarkText">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>249</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #f1f1f1;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>92</y>
        <width>27</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>Text:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_Watermark">
      <property name="geometry">
       <rect>
        <x>147</x>
        <y>10</y>
        <width>142</width>
        <height>22</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"> QComboBox {
     border: 1px solid gray;
     border-radius: 3px;
     padding: 1px 18px 1px 3px;
     min-width: 6em;
    background: #f1f1f1;

 }

 QComboBox:editable {
     background: #f1f1f1;
 }

 QComboBox:!editable, QComboBox::drop-down:editable {
      background: #f1f1f1;
 }

 /* QComboBox gets the &quot;on&quot; state when the popup is open */
 QComboBox:!editable:on, QComboBox::drop-down:editable:on {
     background: #333;
 }

 QComboBox:on { /* shift the text when the popup opens */
     padding-top: 3px;
     padding-left: 4px;
 }

 QComboBox::drop-down {
     subcontrol-origin: padding;
     subcontrol-position: top right;
     width: 15px;
     border-left-width: 1px;
     border-left-color: darkgray;
     border-left-style: solid; /* just a single line */
     border-top-right-radius: 3px; /* same radius as the QComboBox */
     border-bottom-right-radius: 3px;
 }

QComboBox QAbstractItemView
{
background-color: #333;
selection-background-color: rgb(170, 170, 255);
border-radius: 0px;
color: #fff;
font: 14pt;
}</string>
      </property>
      <item>
       <property name="text">
        <string>Bottom Right</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Bottom Left</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Bottom Middle</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Center</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Top Right</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Top Left</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Top Middle</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>111</x>
        <y>39</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>Color:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>147</x>
        <y>37</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #f1f1f1;</string>
      </property>
      <property name="text">
       <string>#ffffff</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>147</x>
        <y>61</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #f1f1f1;</string>
      </property>
      <property name="text">
       <string>12</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>118</x>
        <y>63</y>
        <width>26</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>Size:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_watermarkshadow">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>38</y>
        <width>70</width>
        <height>18</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>Shadow</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>95</x>
        <y>14</y>
        <width>49</width>
        <height>14</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>Location:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_previewimage">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>238</y>
      <width>280</width>
      <height>213</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: #f1f1f1;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>466</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none; color: white;</string>
    </property>
    <property name="text">
     <string>Sub-folder to save in:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_NewFolderName">
    <property name="geometry">
     <rect>
      <x>409</x>
      <y>466</y>
      <width>182</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid black;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(235, 235, 235, 255), stop:1 rgba(255, 255, 255, 255));
padding-left: 1px;</string>
    </property>
    <property name="text">
     <string>resized_images</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_PNG">
    <property name="geometry">
     <rect>
      <x>505</x>
      <y>184</y>
      <width>90</width>
      <height>26</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none;

color: white;</string>
    </property>
    <property name="text">
     <string>Save as PNG</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_JPG">
    <property name="geometry">
     <rect>
      <x>405</x>
      <y>184</y>
      <width>90</width>
      <height>29</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none;

color: white;</string>
    </property>
    <property name="text">
     <string>Save as JPG</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Browse">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>50</y>
      <width>292</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #80CBC4; color: white; font-weight: bold;</string>
    </property>
    <property name="text">
     <string>Browse for directory</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Start">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>501</y>
      <width>280</width>
      <height>48</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #80CBC4; color: white; font-weight: bold;</string>
    </property>
    <property name="text">
     <string>Convert All</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-10</y>
      <width>611</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #1B2327;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>20</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>Blitz Batch Image Resizer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>469</x>
       <y>15</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>This free tool is brought
to you by VanDev.ca</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="listWidget_files">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>80</y>
      <width>292</width>
      <height>469</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background: #f1f1f1;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_Quality">
    <property name="geometry">
     <rect>
      <x>307</x>
      <y>211</y>
      <width>284</width>
      <height>20</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none;</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_percent">
    <property name="geometry">
     <rect>
      <x>354</x>
      <y>184</y>
      <width>30</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none;

color: white;</string>
    </property>
    <property name="text">
     <string>100%</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>187</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Quality:</string>
    </property>
   </widget>
   <zorder>listWidget_files</zorder>
   <zorder>label_previewimage</zorder>
   <zorder>label_7</zorder>
   <zorder>lineEdit_NewFolderName</zorder>
   <zorder>pushButton_Browse</zorder>
   <zorder>pushButton_Start</zorder>
   <zorder>frame</zorder>
   <zorder>tabWidget</zorder>
   <zorder>radioButton_JPG</zorder>
   <zorder>radioButton_PNG</zorder>
   <zorder>slider_Quality</zorder>
   <zorder>label_percent</zorder>
   <zorder>label_10</zorder>
  </widget>
  <action name="actionOpen_directory">
   <property name="text">
    <string>Open directory</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
